src/index.ts

```diff
@@ -1 +1,3 @@
-export { Sdk as default } from 'Sdk';
+import { default as PrezlySdk } from './Sdk';
+
+module.exports = PrezlySdk;
```

Comment:
Why not just Prezly?

Reply:
Changed the export with 8c7c07154f6b1277c07432cd90eb643e628b9473.